applicationinsights/applicationinsights-js.d.ts

```diff
@@ -0,0 +1,488 @@
+// Type definitions for ApplicationInsights-JS v0.22.14
+// Project: https://github.com/Microsoft/ApplicationInsights-JS
+// Definitions by: Kamil Szostak <https://github.com/kamilszostak>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module Microsoft.ApplicationInsights.Context {
+    class Application {
+        /**
+         * The application version.
+         */
+        ver: string;
```

Comment:
I'll change in the SDK and update the definition file in a separate pull request.